Modules/CalcOffence-3_0.lua

```diff
@@ -1599,6 +1602,25 @@ function calcs.offence(env, actor, activeSkill)
 		else
 			output.ChillChanceOnCrit = 100
 		end
+		if skillModList:Flag(cfg, "CritAlwaysAltAilments") and skillFlags.hit then
+			skillFlags.inflictscorch = true
```

Comment:
`inflictscorch` should follow camelCase convention, like so: `inflictScorch`.
Repeat for the other two ailments.